src/applications/vre/28-8832/config/chapters/status-selection/status.js

```diff
@@ -0,0 +1,22 @@
+export const schema = {
+  type: 'object',
+  properties: {
+    status: {
+      type: 'string',
+      enum: ['isActiveDuty', 'isVeteran', 'isSpouse', 'isChild'],
+      enumNames: [
+        'Active duty service member',
+        'Veteran',
+        'Spouse of a Veteran or service member',
+        'Child of a Veteran or service member',
+      ],
+    },
+  },
+};
+
+export const uiSchema = {
+  status: {
```

Comment:
It may need to be in a future pull request, after pre-fill gets wired up, but I think this field needs to have `ui:required` on it.